docs/installation.rst

```diff
@@ -70,7 +69,7 @@ At the command line:
 
 .. code-block:: bash
 
-    $ pip install --user cookiecutter
+    $ pip3 install --user cookiecutter
```

Comment:
I am 100% sure that on modern systems like CentOS 8, pip gets installed as pip3 and non pip. Some applies to virtualenv.

If you want, a more generic command would be `python3 -m pip install ...`, which is probably the most portable example.

pip + virtualenv are special tools which need the version prefix to avoid conflicts with their py3 versions.

Reply:
I can confirm the same behavior on Debian Buster on docker. When installing `python3` and `python3-pip`, there is no `python` or `pip` executables, only `python3` and `pip3`.

Anyway, +1 to `python3 -m pip install`

Reply:
+1 to `python3 -m pip install`